tomopy/prep/normalize.py

```diff
@@ -202,15 +206,22 @@ def normalize_bg(tomo, air=1, ncore=None, nchunk=None):
     """
     tomo = dtype.as_float32(tomo)
     air = dtype.as_int32(air)
+    if out is None:
+        out = tomo.copy()
+
+    axis_size = out.shape[0]
+    ncore, nchunk = mproc.get_ncore_nchunk(axis_size, ncore, nchunk)
 
-    arr = mproc.distribute_jobs(
-        tomo,
-        func=extern.c_normalize_bg,
-        args=(air,),
-        axis=0,
-        ncore=ncore,
-        nchunk=nchunk)
-    return arr
+    chnks = np.round(np.linspace(0, axis_size, ncore+1)).astype(np.int)
+    mulargs = []
+    for i in range(ncore):
+        mulargs.append(extern.c_normalize_bg(out[chnks[i]:chnks[i+1]], air))
+    e = cf.ThreadPoolExecutor(ncore)
```

Comment:
If you are going to use cf, it is probably better to allow the schedule to be passed in (and default to a new thread pool if none is provided).  This will allow something like `dask` to just 'drop in' or to re-use a thread pool across function calls.